### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by lisaMTayl**

### CODE REVIEW

**Comment by lisaMTayl** [comment]
Updated and ready for review!

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,384 @@
...
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/** Container Styles */
```
Comment (AdetunjiShennaike) [inline css/index.css]: i like that you're commenting to separate

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,384 @@
...
+}
+
+.services-content .services-info .facts p {
+    margin-bottom: 20px;
+}
+
+.services-content .services-info .services-info-img {
+    width: 420px;
+    height: 300px;
+}
```
Comment (AdetunjiShennaike) [inline css/index.css]: good class names as well

File: index.html [hunk 3]
```diff
@@ -12,5 +13,76 @@
...
+    <nav>
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </nav>
+    <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+  </header>
```
Comment (AdetunjiShennaike) [inline index.html]: nice separation

File: css/index.css [hunk 4]
```diff
@@ -202,183 +190,153 @@ header .logo {
...
 }
 
-.services-content .services-info .facts p {
-    margin-bottom: 20px;
-}
 
-.services-content .services-info .services-info-img {
-    width: 420px;
-    height: 300px;
-}
```
Comment (AdetunjiShennaike) [inline css/index.css]: the comments dissipated =\

**Review by AdetunjiShennaike — APPROVED** [review]

**Pull request merged**